string_assignment.js

```diff
@@ -0,0 +1,164 @@
+// # String Exercises
+
+let name = "Jaiden";
+// 1. `console.log` the first character of a string.
+console.log(name[0]);
+
+//
+// 2. `console.log` the length of a string.
+console.log(name.length);
+// 3. `console.log` the last character of any string.
+console.log(name[name.length-1]);
+// 4. Create a code block that takes a single string variable, and returns a copy of the string with the last letter capitalized. For example:
+//
+// ```js
+// 'jimmy'
+// // => 'jimmY'
+let newName = "Reione";
+let length = newName.length-1;
+console.log(newName.substring(0,length) + newName.slice(length).toUpperCase());
+
+
+// ```
+//
+// 5. Create a drEvil code block that will take a single number variable, and log the '<variableAmount> dollars',
+// your code block should add '(pinky)' at the end of the amount if it's 1 million. For example:
+//
+// ```js
+// let amount = 10
+// // => 10 dollars
+// let amount = 1000000
+// // =>  1000000 dollars (pinky)
+// ```
+let amount = 300;
+let sentence = `${amount} (pinky)`;
```

Comment:
Good use of string interpolation but this code would not give us the result that we want. For example in this case when amount is 300, it should output "300 dollars" without pinky as amount is not 1 million.